var/spack/repos/builtin/packages/py-sphinx-rtd-dark-mode/package.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PySphinxRtdDarkMode(PythonPackage):
+    """A Sphinx extension for adding a toggleable dark mode to the Read the Docs theme."""
+
+    homepage = "https://github.com/MrDogeBro/sphinx_rtd_dark_mode"
+    pypi = "sphinx-rtd-dark-mode/sphinx_rtd_dark_mode-1.2.4.tar.gz"
+
+    version("1.2.4", sha256="935bc1f3e62fc76eadd7d2760ac7f48bab907a97e44beda749a48a2706aeed63")
+
+    depends_on("py-setuptools", type="build")
+    depends_on("py-sphinx", type=("build", "run"))
```

Comment:
sphinx doesn't seem to be imported in this package

Reply:
`depends_on("py-sphinx", type=("build", "run"))` is not sufficient to pull in sphinx?

Reply:
I'm saying I don't think Sphinx is a dependency

Reply:
ahh, got it.